app/controllers/charge_controller.js

```diff
@@ -50,13 +56,14 @@ const appendChargeForNewView = async function appendChargeForNewView (charge, re
   charge.googlePayGatewayMerchantID = charge.gatewayAccount.gatewayMerchantId
   charge.googlePayRequestMethodData = getGooglePayMethodData({
     allowedCardTypes: supportedNetworksFormattedByProvider(cardModel.allowed, 'google'),
-    merchantId: process.env.GOOGLE_PAY_MERCHANT_ID,
+    merchantId: GOOGLE_PAY_MERCHANT_ID,
     gatewayMerchantId: charge.gatewayAccount.gatewayMerchantId
   })
   charge.googlePayRequestDetails = googlePayDetails
 
   const correlationId = req.headers[CORRELATION_HEADER] || ''
   charge.worldpay3dsFlexDdcJwt = await worlpay3dsFlexService.getDdcJwt(charge, correlationId)
+  charge.worldpay3dsFlexDdcUrl = NODE_ENV !== 'production' ? WORLDPAY_3DS_FLEX_DDC_TEST_URL : WORLDPAY_3DS_FLEX_DDC_LIVE_URL
```

Comment:
I don’t think is quite right. We want to use the test URL if it’s a Worldpay test account and the live URL if it’s a Worldpay live account.

(We manage the difference between environments by having the test and live URL environment variables both point to the test URL in the test environment.)

Reply:
Good point!